src/canister/user_index/src/api/upgrade_individual_user_template/update_user_index_upgrade_user_canisters_with_latest_wasm.rs

```diff
@@ -149,6 +149,7 @@ async fn upgrade_user_canister(
                 configuration.url_to_send_canister_metrics_to.clone(),
             ),
         },
+        true
```

Comment:
Maybe pass `false` just to be on the safe side even if we know that the code doesn't execute right now?

Reply:
Right

Reply:
Done.